<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>newOopsApp - fluent distribution</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>360</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>360</height>
       </size>
      </property>
      <widget class="QWidget" name="page_0">
       <widget class="QGroupBox" name="Auth">
        <property name="geometry">
         <rect>
          <x>29</x>
          <y>110</y>
          <width>231</width>
          <height>141</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="title">
         <string>Authorization</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="line_staff_id">
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
           <property name="placeholderText">
            <string>Your ID Number, ex. 5601</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_password">
           <property name="inputMethodHints">
            <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>Password</string>
           </property>
           <property name="cursorMoveStyle">
            <enum>Qt::VisualMoveStyle</enum>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_signin">
           <property name="text">
            <string>Sign in</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_1">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>261</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="layout_page1">
         <item>
          <widget class="QLabel" name="label_staff_id">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Your ID</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="button_option_proceed">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>320</y>
          <width>88</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(83, 193, 36);</string>
        </property>
        <property name="text">
         <string>Proceed</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>290</y>
          <width>261</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="hlayout_delivery_options">
         <item>
          <widget class="QRadioButton" name="radio_address">
           <property name="text">
            <string>shipping address</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_delivery">
           <property name="text">
            <string>support delivery</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2_support">
       <widget class="QLabel" name="page_2_support_annotation">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>261</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Address delivery option</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_address_courier">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>261</width>
          <height>28</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Enter street and house #</string>
        </property>
       </widget>
       <widget class="QLabel" name="page_2_list_annotation">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>261</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Newspaper list</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>261</width>
          <height>181</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scroll_newsp_list">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>255</width>
           <height>175</height>
          </rect>
         </property>
         <widget class="QListWidget" name="page_2_list_newsp">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>256</width>
            <height>171</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QPushButton" name="button_back_from_p2s">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>330</y>
          <width>88</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(83, 193, 36);</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sendButton_support">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>330</y>
          <width>88</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2_address">
       <widget class="QLabel" name="page_2_address_annotation">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Select where to deliver</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button_back_from_p2d">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>88</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(83, 193, 36);</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QScrollArea" name="scrollArea_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>261</width>
          <height>271</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scroll_newsp_addresses">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>255</width>
           <height>265</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="layout_newsp_addresses">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QPushButton" name="sendButton_address">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>320</y>
          <width>88</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3_report">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>261</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Report</string>
        </property>
       </widget>
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>251</width>
          <height>301</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scroll_report">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>245</width>
           <height>295</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="vlayout_report">
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>298</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
